coalib/testing/LocalBearTestHelper.py

```diff
@@ -52,6 +54,22 @@ class LocalBearTestHelper(unittest.TestCase):
     If you miss some methods, get in contact with us, we'll be happy to help!
     """
 
+    def assertComparableObjectsEqual(self, observed_result, expected_result):
+        self.assertEqual(len(observed_result), len(expected_result),
+                         msg='The number of elements in the observed result '
+                             'and expected result differ.')
```

Comment:
Actually I would fall back to `assertEqual` when the length mismatches. This is the easiest to get this comparative behaviour. We can improve the output messages later for such cases taking `Comparable`-classes into account.